hkm/templates/hkm/views/search_record.html

```diff
@@ -20,7 +20,7 @@
 {% block content %}
 
   {% if search_result %}
-    {% if record.index %}
+    {% if previous_record or next_record %}
```

Comment:
If there's only one result, "back to search" won't be displayed. This should be fixed.

Reply:
Oh damn it, how I didn't think of that 🤦